### PR TITLE
Add Stara credentials documentation

### PR DESCRIPTION
**Pull request by palomacione**

### CODE REVIEW

**Comment by palomacione** [comment]
[[Docs] Add Stara Credentials documentation](https://trello.com/c/5lJAEInK/363-docs-add-stara-credentials-documentation)

**Review by Vinniekun — APPROVED** [review]

File: docs/user_management_endpoints.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 [13]: #get-usersleafuseridagleader-credentials
 [14]: #post-usersleafuseridagleader-credentials
 [15]: #delete-usersleafuseridagleader-credentials
+[16]: #get-usersleafuseridstara-credentials
+[16]: #post-usersleafuseridstara-credentials
+[16]: #delete-usersleafuseridstara-credentials
```
Comment (fabyan-alexander) [inline docs/user_management_endpoints.md:27]: I think the anchor id has to be serial

File: docs/user_management_endpoints.md [hunk 2]
```diff
@@ -346,7 +372,7 @@ Creates a Leaf User.
   "name": "str",
   "email": "help@withleaf.io",
   "phone": "str",
-  "address": "str",
+  "address": "str"
```
Comment (fabyan-alexander) [inline docs/user_management_endpoints.md:375]: Nice catch!

File: docs/user_management_endpoints.md [hunk 3]
```diff
@@ -1430,6 +1456,200 @@ Delete Leaf User's CNHI credentials.
...
+  TOKEN = 'YOUR_TOKEN'
+
+  endpoint = 'https://api.withleaf.io/services/usermanagement/api/users/{leafUserId}/stara-credentials'
+  headers = {'Authorization': f'Bearer {TOKEN}'}
+
+  data = {
+  "apiKey": "str",
+  "accessToken": "str",
+  "accessTokenClient": "str",
+  "refreshToken": "str"
```
Comment (fabyan-alexander) [inline docs/user_management_endpoints.md:1580]: Should we include an indentation group here?

File: docs/user_management_endpoints.md [hunk 4]
```diff
@@ -1430,6 +1456,200 @@ Delete Leaf User's CNHI credentials.
...
+  const TOKEN = 'YOUR_TOKEN'
+
+  const endpoint = 'https://api.withleaf.io/services/usermanagement/api/users/{leafUserId}/stara-credentials'
+  const headers = { 'Authorization': `Bearer ${TOKEN}` }
+
+  const data = {
+  "apiKey": "str",
+  "accessToken": "str",
+  "accessTokenClient": "str",
+  "refreshToken": "str"
```
Comment (fabyan-alexander) [inline docs/user_management_endpoints.md:1557]: Should we include an indentation group here?

**Review by fabyan-alexander — APPROVED** [review]
Great work! I think there are some small points to fix!

File: docs/user_management_endpoints.md (hunk 1 above)
Comment (palomacione) [reply docs/user_management_endpoints.md:27]: yes, copy paste got me

File: docs/user_management_endpoints.md (hunk 4 above)
Comment (palomacione) [reply docs/user_management_endpoints.md]: 8b1074b

File: docs/user_management_endpoints.md (hunk 3 above)
Comment (palomacione) [reply docs/user_management_endpoints.md]: 8b1074b

File: docs/user_management_endpoints.md (hunk 1 above)
Comment (palomacione) [reply docs/user_management_endpoints.md]: 8b1074b

**Review by ricardosantacattarina — APPROVED** [review]
Nice

**Pull request merged**